### DISCUSSION TITLE
Excercise 2 Matrikelnummer

### DISCUSSION THREAD

**Discussion by ghost** [irgroup-classrooms/dis08-2021#17] [Q&A]
Bei der Excercise 2 in Github wo man die Aufgaben nach den letzten 3 Ziffern der Matrikelnummer auswählen soll habe ich eine Frage.
Die letzten drei Ziffern bei meiner Matrikelnummer sind 500.
10-5 = 5
10-0 = 0

Jedoch was soll ich bei der letzten Zahl machen?
Ich habe jetzt einfach eine 1 hoch gezählt und hab somit die Aufgabe 1 abgegeben. Ist dies so in Ordnung?

**Comment by HaakFab** [comment]
Hallo,
Nehmen Sie doch einfach 5, 10 (10-0=10) und 1. Alternativ könnten Sie auch 5, 10 und 11 nehmen also, +1 auf die aufsummierte Zahl. 

VG,
Fabian Haak
